### PR TITLE
[CodeClean] internal function to release svc handle

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Prevent error case if nns-edge dependency does not exists. Add internal function to release ml-svc handle.

### CODE REVIEW

File: c/src/ml-api-service-agent-client.c [hunk 1]
```diff
@@ -519,6 +519,29 @@ ml_service_get_pipeline_state (ml_service_h h, ml_pipeline_state_e * state)
...
+int
+ml_service_pipeline_release_internal (void *priv)
+{
+  _ml_service_server_s *server = (_ml_service_server_s *) priv;
+  g_autoptr (GError) err = NULL;
+  int ret;
+
+  ret = ml_agent_pipeline_destroy (server->id, &err);
+  if (ret < 0) {
+    _ml_error_report_return (ret,
```
Comment (gichan-jang) [inline c/src/ml-api-service-agent-client.c:534]: not `_ml_error_report()` but `_ml_error_report_return`?

Comment (jaeyun-jung) [reply c/src/ml-api-service-agent-client.c:534]: it is internal error case when failed to release pipeline from ml-agent. (previous code also returns error)

**Review by gichan-jang — APPROVED** [review]
LGTM

File: c/src/ml-api-service-agent-client.c [hunk 2]
```diff
@@ -519,6 +519,29 @@ ml_service_get_pipeline_state (ml_service_h h, ml_pipeline_state_e * state)
...
+ * @brief Internal function to release ml-service pipeline data.
+ */
+int
+ml_service_pipeline_release_internal (void *priv)
+{
+  _ml_service_server_s *server = (_ml_service_server_s *) priv;
+  g_autoptr (GError) err = NULL;
+  int ret;
+
+  ret = ml_agent_pipeline_destroy (server->id, &err);
```
Comment (wooksong) [inline c/src/ml-api-service-agent-client.c:532]:
```suggestion
  int ret = ml_agent_pipeline_destroy (server->id, &err);
```

**Review by wooksong — APPROVED** [review]

**Pull request merged**